src/flood_forecaster_cli/commands/database_model.py

```diff
@@ -0,0 +1,50 @@
+"""
+Data modelling Commands
+"""
+
+import click
+from flood_forecaster.utils.configuration import Config
+from flood_forecaster.utils.database_helper import DatabaseConnection
+# from flood_forecaster.utils.logging import logger
```

Comment:
Do we have a logger? It was in my "improvements" list, I think

Reply:
Not yet - in the todo list, yet not a priority.